internal/db/migrations/2023-09-29.0-alter-receiver_wallets-add-invitation_sent_at-column.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+ALTER TABLE
+    public.receiver_wallets
+ADD
+    COLUMN invitation_sent_at timestamp with time zone;
+
+UPDATE
+    public.receiver_wallets
+SET
+    invitation_sent_at = NOW()
+WHERE
+    status = 'REGISTERED';
+    
+-- +migrate Down
+ALTER TABLE
+    public.receiver_wallets DROP COLUMN invitation_sent_at;
```

Comment:
It's not a big deal, but it's a good practice to have an empty line at the end of files.